lte/gateway/c/session_manager/SessionManagerServer.cpp

```diff
@@ -12,6 +12,9 @@
  */
 #include "SessionManagerServer.h"
 #include "magma_logging.h"
+#include <grpc/impl/codegen/port_platform.h>
```

Comment:
I was perhaps lazy on my include sequencing here. I'm not sure what our standards are - if any today - but I plan to fix this with IncludeWhatYouUse soon anyway?

Reply:
These includes may not have been blockers for this building for me - the linker addition was a blocker.  Why this was linking without -lgpr I am unclear.

The new header includes may not be strictly necessary, we may be historically getting them through indirect means. But these are the headers advised for GPR_ASSERT in their documentation (https://grpc.github.io/grpc/cpp/impl_2codegen_2log_8h.html).

We need to scrub all our includes with IncludeWhatYouUse (#5062) as today we rely on luck for many of our header imports (the ordering of compilation and header processing - which has no reason to remain stable across upgrades to CMake / GCC / Clang).

Reply:
Thanks. I think order of compilation is helping. That may be why we have so many issues when we use IDE, but build still works